opencti-platform/opencti-graphql/src/modules/notifier/notifier-statics.ts

```diff
@@ -263,7 +263,7 @@ export const STATIC_NOTIFIERS: Array<BasicStoreEntityNotifier> = [
                                                             </tr>
                                                          </tbody>
                                                       </table>
-                                                      <h2 style="font-size: 18px; font-weight: 400; margin: 0; text-align: center">Automatic digest subscription</h2>
+                                                      <h2 style="font-size: 18px; font-weight: 400; margin: 0; text-align: center">Automatic trigger for participant/Assignee</h2>
```

Comment:
That is not only for participant or assignee I think. We might want to just remove this part.